awscli/clidriver.py

```diff
@@ -197,6 +198,12 @@ def main(self, args=None):
                    '"aws configure".' % e)
             self._show_error(msg)
             return 255
+        except KeyboardInterrupt:
+            # Shell standard for signals that terminate
+            # the process is to return 128 + signum, in this case
+            # SIGINT=2, so we'll have an RC of 130.
+            sys.stdout.write("\n")
+            return 128 + signal.SIGINT
```

Comment:
Because it was an uncaught exception that propagated all the way up the stack, the default python behavior is an RC of 1.